### ISSUE TITLE
No such file or directory: './  _input.p' 

### ISSUE THREAD

**Issue by JappyPing** [GentleZhu/HEER#10]
Hi,  could you please send me or upload the '_input.p' files?
When I was performing HEER according to the instructions, I received FileNotFoundError: [Errno 2] No such file or directory: './HEER/.._input.p'.  Great thanks!

main.py Line 104
`	_network = tdata.TensorDataset(t.LongTensor(pickle.load(open(args.data_dir + args.graph_name + '_input.p'))),
                               t.LongTensor(pickle.load(open(args.data_dir + args.graph_name + '_output.p'))))`

**Comment by GentleZhu** [comment]
Did you download the two compressed data file in the readme? DBLP and YAGO? These files are there.

**Comment by JappyPing** [comment]
> Did you download the two compressed data file in the readme? DBLP and YAGO? These files are there.
Hi, Dr Zhu,
Thanks for your reply~ I re-downloaded the four zip flies (dblp.zip, pretrained_dblp_emb.zip, yago.zip and pretrained_yago_emb.zip) which include these files like dblp.config, dblp_ko_0.1.hin and .txt file, however, there is no '_input.p' files in these fold.  
If you have the '_input.p' files, could you sent it to me?
Thanks ~

**Comment by GentleZhu** [comment]
TL;DR. *input.p is generated in run.sh not from the downloaded data.

Check out run.sh. If there are no "_input.p". Lines below will be executed. 
" echo ${green}===Constructing Training Net===${reset}
if [ ! -e  "$root_dir"/intermediate_data/"$network$SUFFIX" ]; then ... "
which will generate the *input.p.

**Comment by JappyPing** [comment]
> TL;DR. *input.p is generated in run.sh not from the downloaded data.
> 
> Check out run.sh. If there are no "_input.p". Lines below will be executed.
> " echo ${green}===Constructing Training Net===${reset}
> if [ ! -e "$root_dir"/intermediate_data/"$network$SUFFIX" ]; then ... "
> which will generate the *input.p.

Thanks Dr. Zhu. Got it~
